README.md

```diff
@@ -594,6 +594,17 @@ A sample consumer would look as follows:
 class MyConsumer < Deimos::ActiveRecordConsumer
   record_class Widget
 
+  class << self
+    # Optional override, returns true by default.
+    # When this method returns true, a record corresponding to the message
+    # is created/updated.
+    # When this method returns false, message processing is skipped and a
+    # corresponding record will NOT be created/updated.
+    def process_message?(payload)
```

Comment:
Hmm... why is this the only class-level method? Seems confusing.

Reply:
There is no reason for it to be an instance-level method since it only needs access to the `payload` param, but I could change that if you like

Reply:
I agree in general, but thinking about how people are going to implement this, one of the common pitfalls is defining methods on the instance when it should be on the class and vice versa. Ruby doesn't have a good way of saying "hey, you put it in the wrong place", so I'd prefer clarity here.